### PR TITLE
Add semantic tokens for `unified-pipeline` project

### PR DESCRIPTION
**Pull request by vardanbansal-harness**
Remove hardcoded colors from unified pipeline and semantic tokens for it.

### CODE REVIEW

File: packages/playground/src/components/execution/execution-status.tsx [hunk 1]
```diff
@@ -27,16 +27,16 @@ const Badge: React.FC<ExecutionStatusProps & BadgeProps> = props => {
     case ExecutionState.RUNNING:
       return minimal ? (
         <div className="flex items-center gap-1">
-          <div className="w-2 h-2 bg-[rgba(226,155,54,1)] rounded-full" />
+          <div className="w-2 h-2 bg-studio-3 rounded-full" />
```
Comment (abhinavrastogi-harness) [inline packages/playground/src/components/execution/execution-status.tsx:30]: `studio-3` is not a good name. 
we should use naming which indicates what it's used for.. eg. `badge` or `status`

File: packages/unified-pipeline/src/components/Canvas/elements/Nodes/GroupNode/GroupNode.tsx [hunk 2]
```diff
@@ -213,10 +213,10 @@ export default function GroupNode(props: NodeProps<GroupNodeProps>) {
           height
         }}
         className={cx(
-          'flex flex-col items-center justify-between text-xs font-medium leading-3 box-border text-left p-2.5 rounded-lg bg-[rgba(15,16,17,1)] border border-[rgba(48,51,54,1)] border-dashed',
+          'flex flex-col items-center justify-between text-xs font-medium leading-3 box-border text-left p-2.5 rounded-lg bg-studio-6 border border-studio-5 border-dashed',
```
Comment (abhinavrastogi-harness) [inline packages/unified-pipeline/src/components/Canvas/elements/Nodes/GroupNode/GroupNode.tsx:216]: we really shouldn't have 10 unique shades of gray just for the studio.
cant we use existing colors anywhere here?

File: packages/unified-pipeline/src/components/CircleOverlay/CircleOverlay.tsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ const CircleOverlay = (props: CircleOverlayInterface) => {
       {circles.map((circle, index) => (
         <div
           key={index}
-          className={`w-[${circle.size}] h-[${circle.size}] absolute translate-x-[-50%] translate-y-[-50%] rounded-full bg-[rgba(255,255,255,${circle.opacity})]`}
+          className={`w-[${circle.size}] h-[${circle.size}] absolute translate-x-[-50%] translate-y-[-50%] rounded-full bg-white/[${circle.opacity}]`}
```
Comment (abhinavrastogi-harness) [inline packages/unified-pipeline/src/components/CircleOverlay/CircleOverlay.tsx:29]: let's not use `white` as it wont work with inverted theme

File: packages/unified-pipeline/src/icons/Expand.tsx [hunk 4]
```diff
@@ -1,26 +1,14 @@
...
-    {...props}
-  >
-    <rect
-      width={24}
-      height={24}
-      fill="rgba(147, 147, 159, 0.1)"
-      fillOpacity={0.1}
-      rx={4}
-    />
+  <svg xmlns="http://www.w3.org/2000/svg" width={24} height={24} fill={props.color} {...props}>
```
Comment (abhinavrastogi-harness) [inline packages/unified-pipeline/src/icons/Expand.tsx:4]: let's move the icons to the `canary/icons` component

**Review by abhinavrastogi-harness — APPROVED** [review]

File: packages/playground/src/components/execution/execution-status.tsx (hunk 1 above)
Comment (vardanbansal-harness) [reply packages/playground/src/components/execution/execution-status.tsx:30]: Yeah I agree, I've found it challenging to come up with good names, we can discuss how to go about it?

File: packages/unified-pipeline/src/components/Canvas/elements/Nodes/GroupNode/GroupNode.tsx (hunk 2 above)
Comment (vardanbansal-harness) [reply packages/unified-pipeline/src/components/Canvas/elements/Nodes/GroupNode/GroupNode.tsx:216]: Pixelpoint has been using different greys in the design for Pipeline Studio

File: packages/unified-pipeline/src/components/CircleOverlay/CircleOverlay.tsx (hunk 3 above)
Comment (vardanbansal-harness) [reply packages/unified-pipeline/src/components/CircleOverlay/CircleOverlay.tsx:29]: Will fix this, currently this component is not rendered anywhere, just part of code base.

File: packages/unified-pipeline/src/icons/Expand.tsx (hunk 4 above)
Comment (vardanbansal-harness) [reply packages/unified-pipeline/src/icons/Expand.tsx:4]: will do

**Pull request merged**